website/docs/r/entity_tags.html.markdown

```diff
@@ -85,7 +85,7 @@ The following arguments are supported:
 
 All nested `tag` blocks support the following common arguments:
 
-  * `key` - (Required) The tag key.
+  * `key` - (Required) The tag key should not use reserved keys (also known as immutable keys), such as account or accountId. It's recommended to choose a unique and descriptive key that does not conflict with existing reserved keys.
   * `values` - (Required) The tag values.
```

Comment:
* `key` - (Required) The key of the tag.

-> **NOTE:** One should not use reserved (immutable) keys with this resource. It is recommended to choose unique and descriptive keys which do not conflict with existing reserved keys.
  * `values` - (Required) The tag values.